string name
string display_name
bool room # is chat room
bool dm # is direct message
